packages/bitcore-wallet-client/src/lib/api.ts

```diff
@@ -1334,7 +1348,7 @@ export class API extends EventEmitter {
     var hash = t.uncheckedSerialize();
     var args = {
       proposalSignature: Utils.signMessage(
-        hash,
+        hash[0] || hash,
```

Comment:
will need typeof check here

`typeof hash === "string" ? hash : hash[0]`
Or something like that.

Reply:
changed